core/src/main/scala/org/apache/spark/util/ManualClock.scala

```diff
@@ -27,6 +27,7 @@ package org.apache.spark.util
 private[spark] class ManualClock(private var time: Long) extends Clock {
 
   private var _isWaiting = false
+  private var _readyForFirstPeek = false
```

Comment:
I could reproduce this issue as well in a Jenkins run in my PR - https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/3351/consoleFull